src/i18n/locale/en.json

```
@@ -671,7 +677,11 @@
   "components.UserProfile.UserSettings.UserGeneralSettings.displayName": "Display Name",
   "components.UserProfile.UserSettings.UserGeneralSettings.generalsettings": "General Settings",
   "components.UserProfile.UserSettings.UserGeneralSettings.localuser": "Local User",
+  "components.UserProfile.UserSettings.UserGeneralSettings.originallanguage": "Discover Language",
+  "components.UserProfile.UserSettings.UserGeneralSettings.originallanguageTip": "Filter content by original language (this will only apply to the \"Popular\" and \"Upcoming\" categories)",
   "components.UserProfile.UserSettings.UserGeneralSettings.plexuser": "Plex User",
+  "components.UserProfile.UserSettings.UserGeneralSettings.region": "Discover Region",
+  "components.UserProfile.UserSettings.UserGeneralSettings.regionTip": "Filter content by region (this will only apply to the \"Popular\" and \"Upcoming\" categories)",
```

```suggestion
  "components.UserProfile.UserSettings.UserGeneralSettings.originallanguageTip": "Filter content by original language (only applies to \"Popular\" and \"Upcoming\" categories)",
  "components.UserProfile.UserSettings.UserGeneralSettings.plexuser": "Plex User",
  "components.UserProfile.UserSettings.UserGeneralSettings.region": "Discover Region",
  "components.UserProfile.UserSettings.UserGeneralSettings.regionTip": "Filter content by region (only applies to \"Popular\" and \"Upcoming\" categories)",
```
Realized my original suggestion was unnecessarily wordy 😄